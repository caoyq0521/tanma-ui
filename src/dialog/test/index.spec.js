import { mount } from '@vue/test-utils';
import TmDialog from '../dialog.vue';

describe('tmDialog', () => {
  it('render dialog', () => {
    const wrapper = mount(TmDialog);
    expect(wrapper).toMatchSnapshot();
  });
})
  