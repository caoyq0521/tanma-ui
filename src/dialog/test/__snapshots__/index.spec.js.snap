// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tmDialog render dialog 1`] = `
<div class="el-dialog__wrapper" style="display: none;" name="dialog-fade">
  <div role="dialog" aria-modal="true" aria-label="dialog" class="el-dialog tm-dialog" style="margin-top: 15vh; width: 50%;">
    <div class="el-dialog__header"><span class="el-dialog__title"></span><button type="button" aria-label="Close" class="el-dialog__headerbtn"><i class="el-dialog__close el-icon el-icon-close"></i></button></div>
    <!---->
    <!---->
  </div>
</div>
`;
